src/index.tsx

```diff
@@ -54,3 +54,25 @@ async function fetchConfig() {
 fetchConfig().then(() => {
     console.log("Configuration complete. Rendering frontend");
 });
+
+// Add FPS meter for development
+if (process.env.NODE_ENV === "development") {
```

Comment:
I think we should make this optional, and independent of dev mode. We may want to benchmark things in prod builds as well. Can you make this a preference? You don't have to add it to the preference dialog, because we can use a code snippet for those sort of diagnostic things:

```javascript
app.preferences.setPreference("statsPanelEnabled", true);
app.preferences.setPreference("statsPanelMode", 0);

Reply:
ok

Reply:
@kswang1029 are you ok with this solution? I think we can hide debug/diagnostic settings from the preferences dialog

Reply:
👌